docs/docs/cloud/quick_start.md

```diff
@@ -16,29 +24,22 @@ This quick start guide will cover how to build a simple chatbot with LangGraph,
 
     ```python
    from langchain_anthropic import ChatAnthropic
-   from langgraph.graph import END, StateGraph, MessagesState
+   from langchain_community.tools.tavily_search import TavilySearchResults
+   from langgraph.prebuilt import create_react_agent
    
    model = ChatAnthropic(model="claude-3-5-sonnet-20240620")
    
-   graph_workflow = StateGraph(MessagesState)
-   
-   
-   def agent(state: MessagesState):
-       response = model.invoke(state["messages"])
-       return {"messages": [response]}
-   
-   
-   graph_workflow.add_node(agent)
-   graph_workflow.add_edge("agent", END)
-   graph_workflow.set_entry_point("agent")
+   tools = [TavilySearchResults(max_results=2)]
    
-   graph = graph_workflow.compile()
+   graph = create_react_agent(model, tools)
```

Comment:
Should we have some reference link or short blurb on what create_react_agent does?